tools/packaging/kata-deploy/local-build/kata-deploy-binaries.sh

```diff
@@ -173,58 +173,49 @@ install_initrd() {
 }
 
 #Install kernel asset
-install_kernel() {
-	export kernel_version="$(yq r $versions_yaml assets.kernel.version)"
+install_kernel_helper() {
+	local kernel_version_yaml_path="${1}"
+	local kernel_name="${2}"
+	local extra_cmd=${3}
+
+	export kernel_version="$(yq r $versions_yaml ${kernel_version_yaml_path})"
```

Comment:
ah, `$(yq r $versions_yaml FOO` can be replaced by `get_from_kata_deps FOO`. The later function is use throughout the kata-deploy-binaries.sh script

Reply:
Done!